docker/test/fasttest/requirements.txt

```diff
@@ -27,7 +27,7 @@ pandas==1.5.3
 pip==24.1.1
 pipdeptree==2.23.0
 pyparsing==2.4.7
-python-apt==2.4.0+ubuntu3
+# python-apt==2.4.0+ubuntu3
```

Comment:
Probably useless, builds are passing without it.

Reply:
@Felixoid @alesapin maybe you know more?

Reply:
It comes from the older script version https://github.com/ClickHouse/ClickHouse/commit/ce4ddf73364d5e6877917f26a93890195bbe89e0

Yes, it's a system lib version, and can be ignored